### PR TITLE
README: Update as per rename & repo location

### PR DESCRIPTION
**Pull request by zeenix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# docker-machine-kvm
+# docker-machine-libvirt
```
Comment (cfergeau) [inline README.md:1]: :+1:

File: README.md [hunk 2]
```diff
@@ -10,8 +10,8 @@ developed for Docker Machine.
     * Add yourself to the `libvirtd` group (may vary by linux distro) so you don't need to sudo
 * Install [docker-machine](https://github.com/docker/machine/releases)
 * Go to the
-  [releases](https://github.com/dhiltgen/docker-machine-kvm/releases)
-  page and download the docker-machine-driver-kvm binary, putting it
+  [releases](https://github.com/code-ready/machine-driver-libvirt/releases)
+  page and download the docker-machine-driver-libvirt binary, putting it
```
Comment (cfergeau) [inline README.md]: Hmm the whole README is going to be confusing as we don't expect users to download it separately, to run it standalone, and some parts it refers to (boot2iso) are probably not even working at all. Probably better to replace most of the file with some short text saying this is a driver used by CodeReady Containers in order to create/run VMs on linux using libvirt.

**Review by cfergeau — CHANGES_REQUESTED** [review]
I'm afraid you put your hands in a can of worms, you can delete quite a bit of the README file :)

**Comment by zeenix** [comment]
@cfergeau Thanks for the quick review. PTALA. :)

**Review by cfergeau — APPROVED** [review]

**Pull request merged**